examples/oft-solana/tasks/solana/utils.ts

```diff
@@ -0,0 +1,19 @@
+import { Connection, PublicKey } from '@solana/web3.js'
+import { backOff } from 'exponential-backoff'
+
+export const getAccountInfo = async (connection: Connection, publicKey: PublicKey) => {
```

Comment:
I think "getAccountInfo" should be called something else, either "getAccountInfoWithBackoff" or "confirmAccountInitialized"

Reply:
I agree. Additional alternatives include: "assertAccountReadiness", "awaitAccountInfo"..